src/components/Topics.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api.get";
+import { useParams } from "react-router-dom";
+
+export const Topic = () => {
+    const [articles, setArticles] = useState([]);
+    const { topic } = useParams();
+
+    useEffect(() => {
+        getArticles()
+            .then((body) => {
+                setArticles(body.articles);
+            })
+    }, []);
+
+
+    console.log('2.', articles);
+    console.log('4.', topic);
+
+    return (
+        <> 
+        <div className='spacing'></div>
+        
+        <div>{articles.map((article) => {
```

Comment:
I think you would want to be using a filter here, then a map to be explicit.
Is this something that could be handled on your back end?